pyeudiw/satosa/default/openid4vp_backend.py

```diff
@@ -342,6 +340,15 @@ def db_engine(self) -> DBEngine:
 
         return self._db_engine
 
+    def _build_authz_request_url(self, payload: dict) -> str:
+        scheme = self.config["authorization"]["url_scheme"]
+        # NOTE: path component is currently unused by the protocol, but currently
+        # we leave it there as 'authorize' to stress the fact that this is an
+        # OAuth 2.0 request modified by JAR (RFC9101)
+        path = "authorize"
+        query_params = urlencode(payload, quote_via=quote_plus)
+        return f"{scheme}://{path}?{query_params}"
```

Comment:
```suggestion
        return f"{scheme}://{path}?{query_params}"
```

this would work only with custom url schemes, using universal links "://" will be duplicated and wrongly appended

I would suggest something like this

```suggestion
        if "://" in scheme:
            scheme = scheme
        else:
           scheme = f"{scheme}://"
        return f"{scheme}{path}?{query_params}"
```

Reply:
to support univesal link, further changes were added; this includes

1. added to docs that the same configuration can be used for costum scheme or universal link
2. a check for the existance of trailing slash before path component is added (required if universal link does not include them)
3. an extension of same device integration test to make it compatible with universal links, as in that case a different exception is raised